ibis/backends/pyspark/tests/test_null.py

```diff
@@ -4,35 +4,59 @@
 pytestmark = pytest.mark.pyspark
 
 
-def test_isnull(client):
+@pytest.mark.parametrize(
+    'is_null_fn', [lambda t, c: t[c].isnull(), lambda t, c: t[c].notnull()]
+)
+def test_isnull(client, is_null_fn):
     table = client.table('nan_table')
     table_pandas = table.compile().toPandas()
 
     for (col, _) in table_pandas.iteritems():
+        ibis_is_null_expr = is_null_fn(table, col)
         result = (
-            table[table[col].isnull()]
+            table[ibis_is_null_expr]
             .compile()
             .toPandas()
             .reset_index(drop=True)
         )
-        expected = table_pandas[table_pandas[col].isnull()].reset_index(
-            drop=True
-        )
+        result['height'] = result['height'].astype('float64')
+        pandas_is_null_series = is_null_fn(table_pandas, col)
+        expected = table_pandas[pandas_is_null_series].reset_index(drop=True)
         tm.assert_frame_equal(result, expected)
 
 
-def test_notnull(client):
-    table = client.table('nan_table')
+def test_isna(client):
+    table = client.table('nan_table').select(['age', 'height'])
     table_pandas = table.compile().toPandas()
 
     for (col, _) in table_pandas.iteritems():
         result = (
-            table[table[col].notnull()]
+            table[table[col].isnan()]
             .compile()
             .toPandas()
             .reset_index(drop=True)
         )
-        expected = table_pandas[table_pandas[col].notnull()].reset_index(
+        result['height'] = result['height'].astype('float64')
+
+        expected = table_pandas[table_pandas[col].isna()].reset_index(
             drop=True
         )
         tm.assert_frame_equal(result, expected)
+
+
+def test_fillna(client):
```

Comment:
Can you move this test to the shared test?

Reply:
Moved to test_generic.py